components/signup/signup_controller/signup_controller.jsx

```diff
@@ -130,6 +130,8 @@ export default class SignupController extends React.Component {
         let serverError;
         if (err.server_error_id === 'store.sql_user.save.max_accounts.app_error') {
```

Comment:
any reason not to add an `|| err.server_error_id === 'api.team.add_user_to_team_from_invite.guest.app_error'` here?

Reply:
Not really, maybe is more clear and easy to follow this way, but I'm not against do it with the or.